packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -37,18 +37,20 @@ import {
 import styles from './styles.css';
 import {getMeetWidgetProps} from './selector';
 
-import events from './constants';
+import {
+  eventNames,
```

Comment:
Could we do `eventNames as defaultEventNames` so we don't get it confused with `this.eventNames`?